src/test/java/api/loans/scenarios/ChangeDueDateAPITests.java

```diff
@@ -260,6 +263,25 @@ public void changeDueDateNoticeIsSentWhenPolicyIsDefined() {
       hasEmailNoticeProperties(steve.getId(), templateId, matchers)));
   }
 
+  @Test
+  public void dueDateChangedEventIsPublished() {
+    final DateTime newDueDate = dueDate.plus(Period.days(14));
+    changeDueDateFixture.changeDueDate(new ChangeDueDateRequestBuilder()
+      .forLoan(loan.getId())
+      .withDueDate(newDueDate));
+
+    Response response = loansClient.getById(loan.getId());
+    JsonObject updatedLoan = response.getJson();
+
+    List<JsonObject> publishedEvents = Awaitility.await()
+      .atMost(1, TimeUnit.SECONDS)
+      .until(FakePubSub::getPublishedEvents, hasSize(2));
```

Comment:
Similarly to another comment, could a comment be added to explain why there should be two events or that the events are cleared before the loan's due date is changed.

Reply:
Added